docs/samples/tests/xunit/TwoWayBindingTest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Xunit;
+
+namespace Bunit.Docs.Samples;
+
+public class TwoWayBindingTest
+{
+  [Fact]
+  public void Test()
+  {
+    using var ctx = new TestContext();
+    var value = string.Empty;
+
+    ctx.RenderComponent<TwoWayBinding>(parameters =>
+      parameters.Bind(
+        p => p.Value,
+        value,
+        newValue => value = newValue));
+  }
```

Comment:
```suggestion
    var currentValue = string.Empty;

    ctx.RenderComponent<TwoWayBinding>(parameters =>
      parameters.Bind(
        p => p.Value,
        currentValue,
        newValue => currentValue = newValue));
  }
```

I think this might make it easier to follow the example.